### DISCUSSION TITLE
Policy Points in the EDC

### DISCUSSION THREAD

**Discussion by sebbader** [eclipse-edc/Connector#1229] [Ideas]
Usage Control is usually systemised around Policy Points (Policy Information Points, Policy Decision Points etc.), see also [Usage Control in the IDS](https://internationaldataspaces.org/wp-content/uploads/IDSA-Position-Paper-Usage-Control-in-IDS.pdf) page 11 ff or [XACML v3.0](http://docs.oasis-open.org/xacml/3.0/xacml-3.0-core-spec-cs-01-en.pdf) page 19. A defined concept of these - generic - points and how to implement them in the extensions and in particular in downstream applications is needed to implement any flexible Usage Control regime.

**Comment by jimmarino** [comment]
These concepts are already taken into account in the EDC policy design, in particular XACML actors and how different policy models can map to each other (e.g. XACML to/from ODRL).

Is the issue a lack of documentation?
